modules/URL/ConferenceUrl.js

```diff
@@ -45,7 +33,7 @@ export default class ConferenceUrl {
          */
         this.inviteURL
             = location.protocol + "//" + location.host + location.pathname;
-        logger.info("Stored original conference URL: " + this.originalURL);
+        logger.info("Stored original conference URL: " + location.href);
```

Comment:
Should this comment still be here? No storage happened in the constructor. Other commends in this component might need updating too because they reference storing the original conference url.